Bottom: 03e53865cf0a4a2a5cd65d1c148a79c1ab33add8
Top:    fae14a384aa02fc5df6f1f5766ffa4873e57d5e1
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 15:33:28 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 795b9fe..c4b314e 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
